pyfar/io/io.py

```diff
@@ -297,3 +228,210 @@ def write(filename, compress=False, **objs):
 
     with open(filename, 'wb') as f:
         f.write(zip_buffer.getvalue())
+
+
+def read_audio(filename, dtype='float64', **kwargs):
+    """
+    Import an audio file as :py:class:`~pyfar.classes.audio.Signal` object.
+
+    Parameters
+    ----------
+    filename : string, Path
+        Input file.
+    dtype : {'float64', 'float32', 'int32', 'int16'}, optional
+        Data type of the returned signal, by default ``'float64'``.
+        Floating point audio data is typically in the range from
+        ``-1.0`` to ``1.0``.  Integer data is in the range from
+        ``-2**15`` to ``2**15-1`` for ``'int16'`` and from ``-2**31`` to
+        ``2**31-1`` for ``'int32'``.
+    **kwargs
+        Other keyword arguments to be passed to ``soundfile.read``. This is
+        needed, e.g, to read RAW audio files.
+
+
+    Returns
+    -------
+    signal : Signal
+        :py:class:`~pyfar.classes.audio.Signal` object containing the audio
+        data.
+
+    Notes
+    -----
+    * This function is based on ``soundfile.read``.
```

Comment:
Sphinx makes it possible to cross-reference the docs of other packages. Should we do that?

Reply:
I added the cross-references, but I think we need to adjust the sphinx settings accordingly for it to work. I'd suggest to do that in a separate pull request together with the changes at other positions in the code and in CONTRIBUTING.rst.